QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -382,6 +382,14 @@ def convert_to_cloud_kvstyle(
         for output_name in key_value_names:
             fp.write(f" - IOName: {output_name}_RetainedState\n   Precision: float16\n\n")
 
+    with open(os.path.join(onnx_dir_path, "custom_io_int8.yaml"), "w") as fp:
```

Comment:
I feel, there is no overhead to create int8.yaml during export along with fp16.yaml.
Also, do not want to alter existing compile_kv api() since it anyway takes customio as an input. So, user will have to take care of it. But from high level pov, we can keep it ready at ONNX_DIR and pass it from infer/compile with arg "--mxint8".

Reply:
Okay sure!
So, in case, user wants to use pre-exported onnx file and directly uses compile API, this will fail giving `FileNotFoundError`, In this case we should prompt user to rerun `export` in this case?

Reply:
In that case, he will anyway have custom_io_int8.yaml right? `compile_kv_model_on_cloud_ai_100 needs` custom_io_path always. Either via `infer` and `compile`. If he is using `export` and followed by `compile` this should work fine.

Reply:
Updated comment.